src/lib/openapi/spec/api-token-schema.ts

```diff
@@ -6,47 +6,72 @@ export const apiTokenSchema = {
     type: 'object',
     additionalProperties: false,
     required: ['username', 'type'],
+    description: 'An overview of an API token.',
     properties: {
         secret: {
             type: 'string',
+            description: 'The token used for authentication.',
+            example: 'project:environment.xyzrandomstring',
         },
         username: {
             type: 'string',
+            description: 'The name of the token',
+            example: 'some-user',
         },
         type: {
             type: 'string',
             enum: Object.values(ApiTokenType),
+            description: 'The type of API token',
+            example: 'client',
         },
         environment: {
             type: 'string',
+            description: 'The environment the token has access to.',
+            nullable: true,
```

Comment:
Can `environment` be both `null` and `undefined` (not present)?  When is it `null`? And when is it undefined?

Reply:
This question goes for every property that's marked as nullable here.

Reply:
In this particular case I can conclude after having checked the code that no - it's not actually nullable. It's occasionally stored as NULL in the DB - but will get translated to ALL (or '*') when converted to an object. I'll fix